development-defaults.ini

```diff
@@ -1,11 +1,22 @@
 
 debug = True
 ws.auth_required = False
+# set priority plugin so the main uber plugin always loads first
+priority_plugins = uber,
 
 [cherrypy]
 server.socket_host = "0.0.0.0"
 engine.autoreload.on = True
-
+tools.sessions.timeout = 4320  # 30 days (in minutes)
 
 [loggers]
-root = "DEBUG"
\ No newline at end of file
+root = "DEBUG"
+
+[handlers]
+[[stdout]]
+class = "logging.StreamHandler"
+stream = "ext://sys.stderr"
+
+#[[syslog]]
+#class = "logging.handlers.SysLogHandler"
+#address = "/dev/log"
```

Comment:
slight worry about this breaking our logging reporting system.  it might be ok but I think we're relying on it printing to stdout.  our log capture might still capture stderr and stdout, or we may also be overriding it with puppet at the moment so it may not actually matter